src/clr-angular/layout/vertical-nav/vertical-nav-group.ts

```diff
@@ -170,4 +182,17 @@ export class ClrVerticalNavGroup implements AfterContentInit, OnDestroy {
     this._subscriptions.forEach((sub: Subscription) => sub.unsubscribe());
     this._navGroupRegistrationService.unregisterNavGroup();
   }
+
+  /** get all router-links so we could find the active one */
+  @ContentChildren(RouterLinkActive) routerLinks;
+
+  ngAfterContentChecked(): void {
+    if (this.routerLinks.length) {
+      this.routerLinks.forEach(({ isActive, element }) => {
```

Comment:
This is not the right way to add `aria-current="page"` to the active router link. What we need to here is to inject `RouterLinkActive` into `ClrVerticalNavLink` and access its `.isActive` property, e.g:

```ts
@Component({
  selector: '[clrVerticalNavLink]',
  template: `
        ...
    `,
  host: { '[attr.aria-current]': 'routerLinkActive.isActive? "page": null' },
})
export class ClrVerticalNavLink {
  constructor(..., public routerLinkActive: RouterLinkActive) {}

}

Reply:
@Shijir  your solution is looking a lot nicer, but I had a talk with @gnomeontherun and decide that this may require all other approaches on the issue.  And then we could start thinking about how to provide automatically. Right now we are covering only one specific way and we are not sure is everyone using `@angular/router`. So in a way, this is a bit limiting solution. 

Also, I got this idea for adding just a directive to handle this for us.

```html
<a clrAddAriaCurrent>...</a>
```

And based on internal rules clrAddAriaCurrent will add or remove `aria-current` for example

  *   There is `RouterLinkActive` and it is set to `active`
  *   Have class `active` 
  *   Other rules

So, for now, I will just add documentation on the issue and how to handle it by hand.